apps/web/src/payload/payload-types.ts

```diff
@@ -53,6 +110,27 @@ export interface PayloadMigration {
   updatedAt: string;
   createdAt: string;
 }
+export interface Layout {
+  id: number;
+  logo: number | Media;
+  headerItems?:
+    | {
+        type?: ('button' | 'link') | null;
+        title: string;
+        url: string;
+        id?: string | null;
+      }[]
+    | null;
```

Comment:
Probably want to define an interface for both header and footer items. Should make things generally easier on the code side of things